models/lr_test.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+# %pip list
+# get_ipython().system('jupyter nbconvert --to script lr_test.ipynb')
+
+
+# In[2]:
+
+
+# Basic setup
+import numpy as np
+import pandas as pd
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[3]:
+
+
+# %pip install yfinance
+
+
+# In[4]:
+
+
+ls
+
+
+# ## A few reminder examples of plotting and running a script
+
+# In[5]:
+
+
+# %matplotlib inline
+# import numpy as np
+# import matplotlib.pyplot as plt
+
+# t = np.arange(0, 5, 0.2)
+# plt.plot(t, t, 'r--', t, t**2, 'bs', t, t**3, 'g^')
+# plt.show()
+
+
+# In[6]:
+
+
+# %run demo.py
+
+
+# ## Load Nasdaq summary of Symbols
+
+# In[7]:
+
+
+nasdaq = pd.read_csv('nasdaqlisted.txt', delimiter='|')
+print(nasdaq.head())
+
+
+# ## Decide on training and validation sets
+
+# In[48]:
+
+
+Ntotal = 100
+Ntrain = 70
+
+symbols = nasdaq['Symbol'].sample(Ntotal)
+
+symbols_train = list(symbols[:Ntrain].values)
+symbols_val= list(symbols[Ntrain:].values)
```

Comment:
install pylint and use vscode, this will auto fix this type of formatting inconsistencies.